src/azure-cli/azure/cli/command_modules/acs/base_decorator.py

```diff
@@ -42,7 +42,7 @@ def __init__(
         self.__cmd = cmd
         self.__model_module = None
         self.__model_dict = None
-        self.__serializer = None
+        self.__serializer = None  # pylint: disable=unused-private-member
```

Comment:
I checked the code and fount that it's not used anywhere. I think it could be safely removed.

Reply:
```suggestion
```